[0.23 release] add Go to script lists in documentaton (#2477)
kestra-io/docs: content/docs/05.concepts/11.storage.md

### Processing data using scripts

> Kestra can launch scripts written in Python, R, Node.js, Shell, Powershell and Go. Depending on the `runner`, they can run directly in a local process on the host or inside Docker containers.

Reviewer: ```suggestion
Kestra can launch Python, R, Node.js, Shell, Powershell, and Go scripts. Depending on the `runner`, they can run directly in a local process on the host or inside Docker containers.
```

Those script tasks are available in the [Scripts Plugin](https://github.com/kestra-io/plugin-scripts). Below is documentation for each of them:
- The [Python](/plugins/plugin-script-python/io.kestra.plugin.scripts.python.script) task runs a Python script in a Docker container or in a local process.
- The [Node](/plugins/plugin-script-node/io.kestra.plugin.scripts.node.script) task runs a Node.js script in a Docker container or in a local process.
- The [R](/plugins/plugin-script-r/io.kestra.plugin.scripts.r.script) task runs an R script in a Docker container or in a local process.
- The [Shell](/plugins/plugin-script-shell/io.kestra.plugin.scripts.shell.script) task executes a single Shell command, or a list of commands that you provide.
- The [PowerShell](/plugins/plugin-script-powershell/io.kestra.plugin.scripts.powershell.script) task executes a single PowerShell command, or a list of commands that you provide.

> - The [Go (Script)](/plugins/plugin-script-go/io.kestra.plugin.scripts.go.script) task executes a single multi-line script whether the [Go (Commands)](/plugins/plugin-script-go/io.kestra.plugin.scripts.go.commands) task executes a list of commands that you provide.

Reviewer: ```suggestion
- The [Go (Script)](/plugins/plugin-script-go/io.kestra.plugin.scripts.go.script) task executes a single multi-line script, while the [Go (Commands)](/plugins/plugin-script-go/io.kestra.plugin.scripts.go.commands) task executes a list of commands that provide.
```
